sugoi-api-ldap-utils/src/main/java/fr/insee/sugoi/ldap/utils/mapper/GenericLdapMapper.java

```diff
@@ -189,6 +185,11 @@ private static Object transformLdapAttributeToSugoiAttribute(
       case LIST_GROUP:
         values = new ArrayList<>();
```

Comment:
you can simplify these lines by using a flatmap like : 
`        return attrs.stream().flatMap(attribute -> Arrays.stream(attribute.getValues()))
`

Reply:
Done ✅ :) @CChemin